.github/workflows/nrn-modeldb-ci.yaml

```diff
@@ -8,6 +8,29 @@ on:
   schedule:
     # Run at 2am every day
     - cron:  '0 2 * * *'
+  workflow_call:
+   inputs:
+      neuron_v1:
+        description: Last official release (neuron) / NEURON pinned version / Azure drop (artifacts) url
+        default: 'neuron'
+        required: true
+        type: string
+      neuron_v2:
+        description: Last nightly release (neuron-nightly) / NEURON pinned version / Azure drop (artifacts) url
+        default: 'neuron-nightly'
+        required: true
+        type: string
+      models_to_run:
+        description: 'Empty for all models, space separated accession numbers for specific models'
+        default: ''
+        required: false
+        type: string
+      repo:
+        default: 'alexsavulescu/nrn-modeldb-ci'
```

Comment:
Will set this to `neuronsimulator/nrn-modeldb-ci` before merging (currently using this)